app/columns.tsx

```diff
@@ -153,9 +153,9 @@ export const fxmsMessageColumns: ColumnDef<FxMSMessageInfo>[] = [
       </div>
     ),
     cell: (props: any) => {
```

Comment:
I think it might be useful to add some tests in `message-table.test.tsx` to test the correct rendering of the preview buttons for these message surfaces

Reply:
that's a good idea, I'll add some cases.